### ISSUE TITLE
[Feature Request] Possible supporting GLM-5.3-Flash?

### ISSUE THREAD

**Issue by SLAPaper** [FlashML-org/FreeToken#215]
https://huggingface.co/zai-org/GLM-5.3-Flash

320B-A18B

**Comment by loops-and-spells** [comment]
Attempting to do this.  I think KDA, and NoPE are mostly there. mHC 🤷 
Reference logits are obtainable for my blackwells for numerics validation, so I think I can validate. 
🤞

**Comment by loops-and-spells** [comment]
FP8 limits me to 5 layers. I need to use a NVFP4 checkpoint.

**Comment by eduncan911** [comment]
> Attempting to do this. I think KDA, and NoPE are mostly there. mHC 🤷
> Reference logits are obtainable for my blackwells for numerics validation, so I think I can validate.

Could you point us to the instructions to do this for other models?  I've been poking around the wiki and other docs they have; however, I am not seeing the instructions.
